src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -379,6 +379,14 @@ public partial class InvalidMessageContractException : System.Exception
         public InvalidMessageContractException(string message) { }
         public InvalidMessageContractException(string message, System.Exception innerException) { }
     }
+
+    [System.AttributeUsageAttribute((System.AttributeTargets)(384), AllowMultiple = false, Inherited = false)]
+    public class MessageHeaderAttribute : MessageContractMemberAttribute
+    {
+        public bool MustUnderstand { get { return default(bool); } set { } } 
+        public bool Relay { get { return default(bool); } set { } } 
```

Comment:
Are we doing anything with the Relay field? If not, then should we be including it?